_content/excerpts/milarepa-sings-again_larsson-stefan.md

```diff
@@ -0,0 +1,22 @@
+title: "Milarepa Sings Again: Tsangnyön Heruka’s ‘Songs with Parting Instructions’"
+authors:
+  - "Stefan Larsson"
+external_url: "https://doi.org/10.16993/bbi.d"
+drive_links:
+  - "https://drive.google.com/file/d/15bx2xgmDEK7datEJM79Q7aA6LIy1cp8g/view?usp=drive_link"
+course: tibetan
+tags:
+  - milarepa
+  - classical-poetry
+year: 2021
+month: jun
+publisher: "Stockholm University Press"
+address: "Stockholm"
+from_book: songs-on-the-road_larson-af-edholm.md
```

Comment:
It's a slug, so no `.md` at the end: https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_content/papers/travels-and-poems-matsuo-basho_vargo-lars.md?plain=1